search_that_hash/config_object.py

```diff
@@ -10,15 +10,15 @@
 import coloredlogs
 
 
-def cli_config(kwargs):
+def cli_config(kwargs):  # pragma: no cover
```

Comment:
Why is this being ignored

Reply:
I'm not sure how you would test this, but I've removed it